### PR TITLE
[MDCT-3745] Remove unnecessary logout call

### PR DESCRIPTION
**Pull request by BearHanded**
### Description

Users get stuck in a loop if their token is expired when using the Okta login.

Reproducing the issue:
- SEDS refresh tokens are good for 30 days (questionable, should probably change to be in line with the other apps).
- To reproduce this in Dev I currently have it set for 1hr. Returning to that page after 1 hour reproduces the issue.
-I have only been able to reproduce it with Federated logins, because Cognito test user logins is a different auth flow.
We can't use EUA logins in dev branches

The cause:
I believe amplify 4 -> 5 causes the signout call to always navigate to the signout URL where that did not used to be the case (potentially?). When the most recent login with a federated login, the call exhibits this behavior, but not otherwise or on a user that has never logged in (incognito works around this bug).

The solution:
Remove the "convenient" signout calls done on every load of the login page. This clears nothing in the current setup but if the user had used EUA, they get stuck in a loop.

The user store already correctly identifies whether a user is active or not and shows them this page, no further signOut is necessary.

TL;DR:
Remove unnecessary logout call

### Related ticket(s)

CMDCT-3745

---
### How to test

Test that you can log in, and log out as a cognito user, but we can't test EUA until we're in dev.
https://d1dz73cleazm4z.cloudfront.net/

### Notes

---
### Pre-review checklist

- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
- [ ] I have performed a self-review of my code
- [ ] I have manually tested this PR in the deployed cloud environment

---
#### Review
- [ ] Design: This work has been reviewed and approved by design, if necessary
- [ ] Product: This work has been reviewed and approved by product owner, if necessary

#### Security
_If either of the following are true, notify the team's ISSO (Information System Security Officer)._

- [ ] These changes are significant enough to require an update to the SIA.
- [ ] These changes are significant enough to require a penetration test.
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Review by gmrabian — APPROVED** [review]
> SEDS refresh tokens are good for 30 days 

😰

**Review by benmartin-coforma — APPROVED** [review]
